build.Dockerfile

```diff
@@ -4,10 +4,26 @@ RUN go get github.com/tools/godep
 RUN go get github.com/mitchellh/gox
 RUN go get github.com/tcnksm/ghr
 
+# Which docker version to test on
+ENV DOCKER_VERSION 1.6.2
+
+# Download docker
+RUN set -ex; \
+    curl https://get.docker.com/builds/Linux/x86_64/docker-${DOCKER_VERSION} -o /usr/local/bin/docker-${DOCKER_VERSION}; \
+    chmod +x /usr/local/bin/docker-${DOCKER_VERSION}
+
+# Set the default Docker to be run
+RUN ln -s /usr/local/bin/docker-${DOCKER_VERSION} /usr/local/bin/docker
```

Comment:
This is probably not needed anymore (used it to make my early tests on circleci), tell me if you want me to remove it :sweat_smile:.